.storybook/preview.js

```diff
@@ -25,11 +25,7 @@ export const parameters = {
 };
 
 export const decorators = [
```

Comment:
@snowystinger Just a heads up, I know you had previously suggested looking into `layout: centered` as per https://storybook.js.org/docs/react/configure/story-layout. Tried it out, it unfortunately means our stories won't take up the full canvas, hence we would still need a decorator to make the story full screen (aka: adapt to the viewports height + width etc).